west.yml

```diff
@@ -92,7 +92,8 @@ manifest:
       revision: 1080094bc246357c6c35e80e8b2d5d0ce7e0d963
       path: tools/net-tools
     - name: hal_nxp
-      revision: 87850e41cd48e4472adbae03c9623ea349255d48
+      #revision: 87850e41cd48e4472adbae03c9623ea349255d48
+      revision: pull/35/head
```

Comment:
adding `DNM` label until the hal pr gets merged

Reply:
ok